app/src/components/Modals/delete-city-modal.tsx

```diff
@@ -50,9 +50,10 @@ const DeleteCityModal: FC<DeleteCityModalProps> = ({
     formState: { errors, isSubmitting },
     setValue,
   } = useForm<{ password: string }>();
+  let isVoid: void;
   const { t } = useTranslation(lng, "dashboard");
   const [requestPasswordConfirm] = api.useRequestVerificationMutation();
-  const { data: token } = api.useGetVerifcationTokenQuery(null, {
+  const { data: token } = api.useGetVerifcationTokenQuery(isVoid, {
```

Comment:
Just remove this instead of passing isVoid in here (and remove that definition).